Gemfile

```diff
@@ -41,7 +41,9 @@ gem "redcarpet", "~> 3.6"
 # Breadcrumbs!
 gem "gretel", "~> 4.4"
 # Better UI components
+gem "lookbook", ">= 2.0.0.beta.4"
```

Comment:
beta? is that stable?

Reply:
I went with "stable enough" since it's a 2.0 beta.